### ISSUE TITLE
[RA] The official map pool. Altering existing maps, part 2

### ISSUE THREAD

**Issue by SoScared** [OpenRA/OpenRA#11143]
After #10718 I'm extending the alter list into a new issue.

**Forum collaboration thread** 
http://www.sleipnirstuff.com/forum/viewtopic.php?f=82&p=294875#294875

Separate issues will eventually be submitted for each map for a final review and chance for additional discussion.

Map and author names should stay unchanged unless the map is altered a lot beyond it's original structure.

[**Northwest Passage**](http://resource.openra.net/maps/8564/)
Minor aesthetic overhaul
[Altered](http://resource.openra.net/maps/15326/)
**Submitted at #11714**

[**Warwind**](http://resource.openra.net/maps/12545/) (2 players)
Split oil derric access in middle, minor aesthetics
[Altered](http://resource.openra.net/maps/15165/)
**Submitted at #11682**

[**Ghost Town**](http://resource.openra.net/maps/7293/) (4 players)
Grant NE player better access to primary ore patch.
[Altered](http://resource.openra.net/maps/15299/)
**Submitted at #11701**

[**Tabula Rasa**](http://resource.openra.net/maps/6750/) (4 players)
Equalize size of passages through oil derrics' zones.
[Altered](http://resource.openra.net/maps/15301)
**Submitted at #11708**

[**Bombardment Islands**](http://resource.openra.net/maps/8257/) (6 players)
Minor aesthetics.
[Altered](http://resource.openra.net/maps/14749/)
**Submitted at #11530**

[**Doughnut**](http://resource.openra.net/maps/7335/) (6 players)
More ore towards the late game.
Tweak player spawns and build areas. 
[Altered](http://resource.openra.net/maps/13670/)
**Submitted at #11208**

[**Seaside 2**](http://resource.openra.net/maps/13261/) (6 players)
Diversify trees, debris.
Minor aesthetic overhaul.
[Altered](http://resource.openra.net/maps/15228/)
**Submitted at #11683**

[**Snow Town**](http://resource.openra.net/maps/8227/) (6 players) 
Minor aesthetic overhaul.
Tweaked and balanced ridges and shorelines.
[Altered](http://resource.openra.net/maps/15862/)
**Submitted at #11700**

[**Snowy Island**](http://resource.openra.net/maps/7617/) - (6 players)
Tweak shorelines, ridges, debris.
Minor aesthetic overhaul.
[Altered](http://resource.openra.net/maps/13121/)
**Submitted at #11681**

[**Synergy**](http://resource.openra.net/maps/7485/) (6 players)
Structural and aesthetic overhaul.
[Altered](http://resource.openra.net/maps/15320/)
**Submitted at #11713**

[**Center of Attention Redux 2**](http://resource.openra.net/maps/7720/) - (8 players)
Minor aesthetics.
[Altered](http://resource.openra.net/maps/15861/)
**Submitted at #11529** 

[**Coastal Influence**](http://resource.openra.net/maps/7624/) - (8 players)
Minor aesthetics.
Additional ore mines at player spawns.
[Altered](http://resource.openra.net/maps/15321/)
**Submitted at #11711**

[**Alaska Anarchy Redux**](http://resource.openra.net/maps/7992/) - (10 players)
Minor aesthetics.
[Altered](http://resource.openra.net/maps/12800/)
**Submitted at #11209**

[**Engagement**](http://resource.openra.net/maps/7377/) - (10 players)
Minor aesthetics.
Tweak south team's middle ore advantage.
[Altered](http://resource.openra.net/maps/12050/)
**Submitted at #11210**

[**Tainted Peak**](http://resource.openra.net/maps/8088/) - (10 players)
Move island north to equal sea size.
Tweak ore patches, minor aesthetics.
[Altered](http://resource.openra.net/maps/15302/)
**Submitted at #11709**

[**Chaos Canyon**](http://resource.openra.net/maps/7989/) - (12 players)
Minor aesthetics.
Add 4x2 ore mines in corners.
[Altered](http://resource.openra.net/maps/15303/)
**Submitted at #11710**

**Comment by epicelite** [comment]
On Bombardment islands the ore mines need to be evened up(one island has more mines, bottom I think) and the tumors cliffs/plateau in the middle of the top islands need to be removed so build space is even.

**Comment by SoScared** [comment]
@epicelite this ties in to the dilemma with altering maps beyond their original structure. Bombardment Islands is already a very unbalanced map in the ways you said. I'm not gonna attempt to perfectly balance the sides but the ore mine differences is probably a simple fix and there's probably ways to give north a better build area.

**Comment by SoScared** [comment]
Reduced the list due to more maps falling below salvageble. Recently there's been an influx of more quality team maps to the resource center after NoobMapMaker's Map Contest.

**Comment by SoScared** [comment]
I'll eventually be able to submit this as a pull request. For consistency sake this will remain an issue ticket until all individual map tasks has been submitted.

**Comment by SoScared** [comment]
List and map revision links updated.
